src/app/components/case/CaseDetails/CaseDetails.tsx

```diff
@@ -25,9 +25,9 @@ const StyledDiv = styled.div`
   }
 `
 
-const getDataFirstCol = (caseItem?: Components.Schemas.Case) => {
+const getDataFirstCol = (isClosed: boolean, caseItem?: Components.Schemas.Case) => {
   if (caseItem === undefined) return
-  const { id, theme, start_date, sensitive, previous_case, is_enforcement_request } = caseItem
+  const { id, start_date, sensitive, previous_case, is_enforcement_request } = caseItem
   const data: any = {
```

Comment:
Ik weet dat dit niet onderdeel is van deze PR changes, maar moet hier geen interface van zijn. Data is een prop van `<DefinitionList>`. Ik zou verwachten dat deze een interface exporteert, zodat je weet welke variabelen in dit object moet zetten. Zou dit in het kader van de Boys Scout rule meegenomen kunnen worden?

Reply:
Good point!